[GCI-177]: Remove Unnecessary Parentheses in GZIPResponseStream.java

billxu1401 opened the pull request:
<!--- Add a pull request title above in this format -->
<!--- real example: 'TRUNK-5111 Replace use of deprecated isVoided' -->
<!--- 'TRUNK-JiraIssueNumber JiraIssueTitle' -->
Deleted Unnecessary Parentheses in GZIPResponseStream.java
<!--- Describe your changes in detail -->
<!--- It can simply be your commit message, which you must have -->


## Issue I worked on
<!--- This project only accepts pull requests related to open issues -->
<!--- Want a new feature or change? Discuss it in an issue first -->
<!--- Found a bug? Point us to the issue/or create one so we can reproduce it -->
<!--- Just add the issue number at the end: -->
see https://issues.openmrs.org/browse/GCI-177

## Checklist: I completed these to help reviewers :)
<!--- Put an `x` in the box if you did the task -->
<!--- If you forgot a task please follow the instructions below -->
- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

cnagadya commented:
@billxu1401, this PR seems to contain more changes than the issue required ie removing useless parentheses in RZIPResponseStream.java. I notice two other files have been modified. Any reason for this?

billxu1401 commented:
@cnagadya , when I tried to change the commit name on my old PR on this task using git commit --amend -m, the commit changed to a random commit from another guy, and I had no idea why it happened, so I created this new PR. However, 2 unrelated commits were also added when I created this PR, and I don't know why. Since I don't know how to delete the commits, I just squashed them together and renamed the commit. It would be really helpful if you can tell me how to delete the commits. Thanks!

dkayiwa commented:
Can you squash the commits into one as advised at? https://wiki.openmrs.org/display/docs/Pull+Request+Tips
